services/frontend-v3/src/routes/Admin.js

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
 import Keycloak from "keycloak-js";
 import { Route, Redirect } from "react-router-dom";
-import { createBrowserHistory } from "history";
+// import { createBrowserHistory } from "history";
```

Comment:
remove if commented

Reply:
That is supposed to be used by another code piece that is commented out in the same file, so in-order to get rid of the console warning, I decided to comment it out. It is important to keep.

Reply:
However, no more console warnings from this file.